<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLCDNumber" name="lcdNumber">
         <property name="smallDecimalPoint">
          <bool>false</bool>
         </property>
         <property name="numDigits">
          <number>1</number>
         </property>
         <property name="mode">
          <enum>QLCDNumber::Hex</enum>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
         <property name="value" stdset="0">
          <double>16.000000000000000</double>
         </property>
         <property name="intValue" stdset="0">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_2">
         <property name="smallDecimalPoint">
          <bool>false</bool>
         </property>
         <property name="numDigits">
          <number>1</number>
         </property>
         <property name="mode">
          <enum>QLCDNumber::Hex</enum>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
         <property name="value" stdset="0">
          <double>16.000000000000000</double>
         </property>
         <property name="intValue" stdset="0">
          <number>16</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QDial" name="dial">
         <property name="maximum">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="dial_2">
         <property name="maximum">
          <number>16</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>235</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="led_row0_3">
         <item>
          <widget class="QLabel" name="row0_16">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="assets.qrc">:/images/assets/led_red_off.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="row0_17">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="assets.qrc">:/images/assets/led_red_off.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="row0_18">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="assets.qrc">:/images/assets/led_red_off.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="row0_19">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="assets.qrc">:/images/assets/led_red_off.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="row0_20">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="assets.qrc">:/images/assets/led_red_off.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="row0_21">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="assets.qrc">:/images/assets/led_red_off.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="row0_22">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="assets.qrc">:/images/assets/led_red_off.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="row0_23">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="assets.qrc">:/images/assets/led_red_off.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="led_row0_2">
         <item>
          <widget class="QLabel" name="row0_8">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="assets.qrc">:/images/assets/led_red_off.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="row0_9">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="assets.qrc">:/images/assets/led_red_off.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="row0_10">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="assets.qrc">:/images/assets/led_red_off.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="row0_11">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="assets.qrc">:/images/assets/led_red_off.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="row0_12">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="assets.qrc">:/images/assets/led_red_off.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="row0_13">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="assets.qrc">:/images/assets/led_red_off.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="row0_14">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="assets.qrc">:/images/assets/led_red_off.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="row0_15">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="assets.qrc">:/images/assets/led_red_off.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="led_row0">
         <item>
          <widget class="QLabel" name="row0_7">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="assets.qrc">:/images/assets/led_red_off.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="row0_6">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="assets.qrc">:/images/assets/led_red_off.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="row0_5">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="assets.qrc">:/images/assets/led_red_off.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="row0_4">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="assets.qrc">:/images/assets/led_red_off.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="row0_3">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="assets.qrc">:/images/assets/led_red_off.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="row0_2">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="assets.qrc">:/images/assets/led_red_off.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="row0_1">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="assets.qrc">:/images/assets/led_red_off.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="row0_0">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="assets.qrc">:/images/assets/led_red_off.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="button_goto">
         <property name="text">
          <string>GOTO</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="button_goto_2">
         <property name="text">
          <string>POKE</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="button_goto_3">
         <property name="text">
          <string>PEEK</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="button_goto_4">
         <property name="text">
          <string>DEPOSIT</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>418</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
